### PR TITLE
[WIP] NetworkManager: update to 1.16.0.

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

**Comment by Cogitri** [comment]
Still needs cross gir fixed.

File: srcpkgs/NetworkManager/patches/fix-cross-gir.patch [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
++++ libnm/meson.build
+@@ -244,7 +244,23 @@ if enable_introspection
+     )
+   endif
+ 
+-  generate_setting_docs = join_paths(meson.current_source_dir(), 'generate-setting-docs.py')
++  if meson.is_cross_build()
++      arch = host_machine.cpu_family()
++      if arch == 'x86'
++              qemu_arch = 'ia32'
```
Comment (maxice8) [inline srcpkgs/NetworkManager/patches/fix-cross-gir.patch]: There is no ia32, you are looking for i386

Comment (Cogitri) [reply srcpkgs/NetworkManager/patches/fix-cross-gir.patch]: Oops, copied that from fwupd

**Comment by Cogitri** [comment]
Seems like nss on musl doesn't build right now.

**Comment by Cogitri** [comment]
Let's see if Travis like this

**Pull request merged**